experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -1222,4 +1223,22 @@ describe('fetch', () => {
       );
     });
   });
+
+  describe('when request headers are applied as attributes', () => {
+    afterEach(() => {
+      clearData();
+    });
+
+    it('applies request headers supplied in config', async () => {
+      await prepareData(url, () => getData(url), {
+        requestHeadersAsAttributes: ['foo', 'Content-Type'],
+      });
+      const span: tracing.ReadableSpan = exportSpy.args[1][0][0];
+      assert.ok(span.attributes[ATTR_HTTP_REQUEST_HEADER('foo')] === 'bar');
+      assert.ok(
+        span.attributes[ATTR_HTTP_REQUEST_HEADER('Content-Type')] ===
```

Comment:
The spec expects "normalized HTTP Header name (lowercase)" so this test is a bit misleading/wrong